### ISSUE TITLE
Version 1.2.2  useEnhancedFanOut causing error 

### ISSUE THREAD

**Issue by mikedizon** [lifion/lifion-kinesis#430]
```
const kinesis = new Kinesis({
    consumerName: 'consumerName',
    streamName: 'streamName',
    useEnhancedFanOut: true,
  });
```
Setting `useEnhancedFanOut` to true throws the following error. 

```
UnhandledPromiseRejectionWarning: Error: `input` must not start with a slash when using `prefixUrl`
at normalizeArguments (/app/node_modules/got/dist/source/core/index.js:478:23)
at got (/app/node_modules/got/dist/source/create.js:112:39)
at Function.got.stream (/app/node_modules/got/dist/source/create.js:221:37)
at FanOutConsumer.start (/app/node_modules/lifion-kinesis/lib/fan-out-consumer.js:362:33)
at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Comment by mikedizon** [comment]
It looks like a fix for this was merged into the develop branch three months ago. Is there a timeline for this to be merged into the next release?

**Comment by mikedizon** [comment]
This occurs with both 1.2.2 and the develop branch.

**Comment by eaviles** [comment]
Thanks for your patience @mikedizon! I just published version 1.3.1 with a fix for the issue.
